### ISSUE TITLE
CSV conversion to JSON splits string on : to become a subfield.

### ISSUE THREAD

**Issue by chrisguest75** [mikefarah/yq#1947]
When transforming CSV strings to JSON objects.  Strings that contains : will be split into a field. 
With the field name being the LHS of the : and the value being the RHS.  

Version of yq: 4.40.4
Operating system: WSL using linxbrew
Installed via: homebrew

**Input CSV**
```
id,mystring
1,hello world
2,hello: world
```

**Command**
The command you ran:
```
yq -o=json ./csv/bug2.csv 
```

**Actual behavior**

```yaml
[
  {
    "id": 1,
    "mystring": "hello world"
  },
  {
    "id": 2,
    "mystring": {
      "hello": "world"
    }
  }
]
```

**Expected behavior**

```yaml
[
  {
    "id": 1,
    "mystring": "hello world"
  },
  {
    "id": 2,
    "mystring":"hello: world"
    }
  }
]
```

**Additional context**

It also fails when using quoted strings. 

```
"id","mystring"
"1","hello world"
"2","hello: world"
```

**Comment by mikefarah** [comment]
That's a good point, I'll add a new flag in the next release to turn off auto-parsing. It'll be on by default for backwards compatibility.

**Comment by mikefarah** [comment]
Fixed in 4.41.1, you'll need to pass in the new flag `--csv-auto-parse=f`, as shown here: https://mikefarah.gitbook.io/yq/usage/csv-tsv#parse-csv-into-an-array-of-objects-no-auto-parsing
